app/models/favorite_item.rb

```diff
@@ -1,9 +1,14 @@
 class FavoriteItem < ActiveRecord::Base
+  include Scheduleable
+
   enum favorite_type: { "my_favorite" => 0, "wish_list" => 1 }
+  enum schedule_type: { notify_type: "notify_list_item", delete_type: "delete_list_item" }
```

Comment:
what does "notify_list_item"  "delete_list_item" mean ?  Should be "notify_wish_list_item" "delete_wish_list_item"  ?